galsim/real.py

```diff
@@ -509,6 +559,7 @@ def close(self):
         self.loaded_files = {}
 
     def getNObjects(self) : return self.nobjects
+    def __len__(self): return self.nobjects
```

Comment:
This line is not covered.  It makes sense as a thing for a RealGalaxyCatalog to have, but I guess if you used it for something, you ended up removing that use.  Could probably add a simple test in the main rgc tests that `len(rgc) == rgc.getNobjects() == rgc.nobjects`.